aldryn_newsblog/cms_plugins.py

```diff
@@ -24,7 +24,7 @@ def get_render_template(self, context, instance, placeholder):
 
 
 class NewsBlogPlugin(TemplatePrefixMixin, CMSPluginBase):
-    module = 'NewsBlog'
+    module = 'News & Blog'
```

Comment:
yes it only outputs the name here:
![image](https://cloud.githubusercontent.com/assets/270307/8232121/8de63816-15cd-11e5-8007-13e1e6d57c0d.png)